Makefile

```diff
@@ -395,6 +412,9 @@ images/rootfs-%.yml.in: images/rootfs.yml.in FORCE
 	@sed -e 's#EVE_VERSION#$(ROOTFS_VERSION)-$*-$(ZARCH)#' < $@.sed > $@ || rm $@ $@.sed
 	@rm $@.sed
 
+$(ROOTFS_FULL_NAME)-adam-kvm-$(ZARCH).$(ROOTFS_FORMAT): $(ROOTFS_FULL_NAME)-kvm-adam-$(ZARCH).$(ROOTFS_FORMAT)
```

Comment:
adam-kvm and kvm-adam?? Why two which such similar names?

Reply:
Precisely because they are similar ;-) IOW, I don't want to remember which way it goes -- so I made sure both ways of spelling it are accepted (can we call one BE and the other one AE?).